projects/documentation/content/_includes/partials/demo.njk

```diff
@@ -40,8 +40,7 @@
             horizontal
             href="https://www.npmjs.com/package/@spectrum-web-components/button"
         >
-            <h2 slot="heading">NPM</h2>
-            <h3 slot="subheading">{% packageVersion componentPackage %}</h3>
+            <p slot="heading">NPM {% packageVersion componentPackage %}</p>
```

Comment:
We can't open these cards in a new tab, and middle-click doesn't work because the pointer-events make the text selectable. If there's no easy fix (e.g., opening them by default in a new tab would already help alleviate my concerns), consider this out of scope for this PR. However, let's make sure to address it at a later time!

Reply:
I agree. We should raise this as a separate issue.